app/Views/Admin/Template.php

```diff
@@ -10,8 +10,8 @@
         <meta content="" property="og:image">
         <meta content="A simple database administration tool for simple CRUD operations." property="og:description">
         <meta content="https://github.com/sitilge/curdle" property="og:url">
-        <link rel="stylesheet" href="/css/all.css">
-        <script src="/js/all.js"></script>
+        <link rel="stylesheet" href="<?=$url->admin()?>css/all.css">
+        <script src="<?=$url->admin()?>js/all.js"></script>
```

Comment:
works for me.

It may be worth noting that while short tags "<?" are much-maligned (for good reason), echo tags "<?=" are now universally available, regardless of the short_open_tags php.ini setting
http://php.net/manual/en/migration54.new-features.php